telemetry/definitions/commonDefinitions.json

```diff
@@ -142,6 +147,63 @@
             "name": "aws_reportPluginIssue",
             "description": "Report an issue with the plugin"
         },
+        {
+            "name": "cloudwatchlogs_openGroup",
+            "description": "Open the CloudWatch Logs group window",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "cloudwatchlogs_openStream",
+            "description": "Open a CloudWatch Logs stream in the window",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_downloadToFile",
+            "unit": "Bytes",
+            "description": "Download a log stream to a file. Value indicates the final size of the formatted log stream.",
+            "metadata": [{"type": "result"}]
```

Comment:
Curious, you say the result here refers to the size. Do results of other metrics refer to whether the operation was a success?

If so, is there a way to indicate success/failure of the operation independently from the size here? i.e. the download succeeds/fails

Reply:
Success comes from `result` but size comes from `Value` which along with `starttime` are two things that can always be set independent of everything else